README.md

```diff
@@ -90,6 +91,24 @@ Executables here will run just before those in `/opt/odoo/common/build.d`.
 Files here will be environment-variable-expanded and concatenated in
 `/opt/odoo/auto/odoo.conf` at build time.
 
+#### `/opt/odoo/custom/ssh.d`
```

Comment:
The `.d` suffix in this project normally indicates that things inside it will be looped, either by `run-parts` or concatenanting all of them, or whatever.

This case is different, so just call it `ssh`.